### PR TITLE
Native YAML - cloud/misc

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
cloud/misc/*

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Fale for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/misc/proxmox.py [hunk 1]
```diff
@@ -174,39 +174,116 @@
 author: "Sergei Antipov @UnderGreen"
 '''
 
-EXAMPLES = '''
+EXAMPLES :  '''
```
Comment (gundalow) [inline cloud/misc/proxmox.py]: Looks like there is an extra space in there
```
2016-12-01 17:25:09 + python2.6 -m compileall -fq .
2016-12-01 17:25:13 Compiling ./cloud/misc/proxmox.py ...
2016-12-01 17:25:13 SyntaxError: ('invalid syntax', ('./cloud/misc/proxmox.py', 177, 10, "EXAMPLES :  '''\n"))
```

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

**Comment by gregdek** [comment]
Thanks @Fale for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Review by gundalow — APPROVED** [review]

**Pull request merged**